idea/api-gateway/src/common/genesis-hash-map.ts

```diff
@@ -0,0 +1,3 @@
+const genesisHashMap = new Map<string, string>();
```

Comment:
1. What is the point to keep this and scheduler in common package if it used only in api-gateway?
2. Maybe it's better to use Set instead Map?

Reply:
@osipov-mit What do mean by "if it used only in `api-gateway` ?

Currently `genesisHashMap` declared in `api-gateway`. Each of the services also can have a `common` folder for sharing data inside service no between services.

I think so with you. It will be better to use `Set` instead of `Map`.

Reply:
Ah, yeah, sorry. I misunderstood